MONEY/NetworkEarnBeachPartyLostFound.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: MONEY
+aliases: ["0xA51338E0DCCD4065", "_NETWORK_EARN_FROM_BEACH_PARTY_LOST_FOUND"]
```

Comment:
Same thing here, there is a `_NETWORK_EARN_FROM_BEACH_PARTY_LOST_FOUND` alias that is not present in the previous native documentation anywhere🤔